tests/test_dataset_wrapper.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.model_selection import train_test_split
+
+from datasets.dataset_wrapper import DatasetWrapper
+
+wine_data_path = "../datasets/winequality.csv"
+random_state, test_size = 0, 0.2
+
+# test invalid initailization of DatasetWrapper object
+def test_fie_does_not_exist():
+    with pytest.raises(FileNotFoundError):
+        DatasetWrapper("datasets/winequalit.csv")
```

Comment:
I think this and the next test assumes a different working dir than `wine_path_data` above?

Reply:
Right. I changed this to using the reference path as well.